src/rest/SearchEndpoint.ts

```diff
@@ -1097,3 +1104,66 @@ function includeActionsHistory(historyStore: CoveoAnalytics.HistoryStore = new h
     return descriptor;
   };
 }
+
+function includeReferrer() {
```

Comment:
We don't need to add this parameter, it is already available in the HTTP headers.